src/include/multipath/complex.rb

```diff
@@ -49,7 +50,9 @@ def initialize_multipath_complex(include_target)
       @service_status = 0
       @has_dumbtab = false
       @device_template = "vendor %1; product %2"
-      @builtin_multipath_conf_path = "/tmp/.yast2-multipath-builtin-conf"
+
+      r = SecureRandom.hex
+      @builtin_multipath_conf_path = "/tmp/.yast2-multipath-builtin-conf-" + r[0,8]
```

Comment:
This does not guarantee that the file already exists (and is a nasty symlink pointing to some sensitive place).

Please use Ruby [Tempfile](https://ruby-doc.org/stdlib-2.5.1/libdoc/tempfile/rdoc/Tempfile.html) for creating temporary files, it ensures the file is unique and creates it in a secure way (using `File::CREAT|File::EXCL` flags). Also see the "Good practices" section there and https://yastgithubio.readthedocs.io/en/latest/security-tips/#temporary-files.

Reply:
Thanks for the information. 
I change the 'SecureRandom" to "Tempfile" to create file in /tmp